src/App.js

```diff
@@ -1,14 +1,33 @@
+import React from 'react';
+import {
+  BrowserRouter, NavLink, Routes, Route,
+} from 'react-router-dom';
 import './App.css';
-import Calculator from './components/Calculator';
+import Home from './pages/Home';
+import CalculatorPage from './pages/CalculatorPage';
 import Quote from './components/FetchQuote';
 
-function App() {
+export default function App() {
+  const activeClassName = 'nav-active';
   return (
-    <div className="App">
-      <Calculator />
-      <Quote />
-    </div>
+    <BrowserRouter>
+
+      <header>
+        <h1>Math Magicians</h1>
+        <nav>
+          <NavLink to="home" className={({ isActive }) => (isActive ? activeClassName : undefined)}>Home </NavLink>
+          <li>| </li>
+          <NavLink to="calculator" className={({ isActive }) => (isActive ? activeClassName : undefined)}> Calculator</NavLink>
+          <li>| </li>
+          <NavLink to="quote" className={({ isActive }) => (isActive ? activeClassName : undefined)}>Quote</NavLink>
+        </nav>
+      </header>
+      <Routes>
+        <Route path="/home" element={<Home />} />
+        <Route path="/calculator" element={<CalculatorPage />} />
+        <Route path="/quote" element={<Quote />} />
+      </Routes>
```

Comment:
- [OPTIONAL BUT HIGHLY RECOMMENDED]  It will be nice if you add a default route when the page loads for the first time, as it is right now there is no default route on your website. A default route ensures that users are directed to a meaningful and user-friendly page when they enter your website's base URL (e.g., https://example.com/). Without a default route, users might encounter a blank or error page, which can be confusing and frustrating. Please resolve that :slightly_smiling_face:

Reply:
@Alusp I just made the home page load inmediately as you recommended. Thank you.🥳

Reply:
You are most welcome! @eamador1 :slightly_smiling_face: